numpy/lib/_function_base_impl.py

```diff
@@ -1867,62 +1867,130 @@ def sort_complex(a):
         return b
 
 
-def _trim_zeros(filt, trim=None):
+def _arg_trim_zeros(filt):
```

Comment:
Made this one private for now. Not sure if this function is valuable enough to increase the API surface.